### DISCUSSION TITLE
Funding rate

### DISCUSSION THREAD

**Discussion by jooray** [get10101/10101#1306] [General]
How does the funding rate work? I guess if you are rolling over the positions, the funding rate is included in the difference between future vs. spot. Could this be displayed on the user interface? It is good to understand this to spot arbitrage opportunities and attract market makers.

**Comment by bonomat** [comment]
Funding rate is a complex topic. Let's first get on the same page: 

> The funding rates represent regular payments exchanged between short and long traders who maintain perpetual contract positions. These funding rates exist because perpetual futures contracts, as their name implies, can be held indefinitely without a set expiration date.
> Since perpetual futures contracts never settle in the traditional sense, we need a mechanism to ensure that futures prices and index prices converge on a regular basis. This mechanism is known as Funding Rate.

Our idea is to have weekly expiring contracts (expiring on Sunday, 15:00 GMT) which roll-over if you come on-line between Friday, 15:00 GMT and Sunday, 15:00 GMT. 
The funding rate will expire in this window (TBD) and is paid retrospectively, i.e. if you have a position open at this event, you either have to pay or receive the funding rate. 

The funding rate should be shown in the frontend as well as on your position so that you can clearly see what's going to be charged.
